compiler/damlc/tests/daml-test-files/ExceptionSyntax.daml

```diff
@@ -2,12 +2,13 @@
 -- SPDX-License-Identifier: Apache-2.0
 
 -- @SINCE-LF-FEATURE DAML_EXCEPTIONS
--- @QUERY-LF [ .modules[].exceptions[] ] | length == 1
+-- -- @QUERY-LF [ .modules[].exceptions[] ] | length == 1
```

Comment:
hm I assume this broke because the test now brings in `Daml.Script` and with it more exceptions. I think it should be possible to tweak the jq query with some of the functions from [`compiler/damlc/tests/src/query-lf.jq`](https://github.com/digital-asset/daml/blob/014b0ccc3691b7dc80893430ba445d315f90c7b2/compiler/damlc/tests/src/query-lf.jq), but honestly these are a bit of a pain - you need to get your hands on the `.json` version of the resulting dar, which afaik you can only do by tweaking the target file [here](https://github.com/digital-asset/daml/blob/014b0ccc3691b7dc80893430ba445d315f90c7b2/compiler/damlc/tests/src/DA/Test/DamlcIntegration.hs#L468-L470), so for this particular test case I'd be happy to remove this while adding an `.EXPECTED.rst` that shows the exception declared here.

Reply:
Oh! I completely forgot I commented this out - I'll see if the first option is viable, if not, expected.rst it is

Reply:
Theres also expected.json that I could use, is that the same json file?

Reply:
Oh, but ofc the json file will change as the Daml.Script version changes, so .json files aren't safe here

Reply:
I would argue that this is actually quite limiting, and that a mechanism to omit Script versions, or even explicitly expect the correct one via some kind of pre-defined constant (like ghc_version but in the golden file) would allow these test to be more usable

Reply:
Turns out i must have removed it to check some stuff, it works fine with it uncommented, added back in